dashboard/app/controllers/projects_controller.rb

```diff
@@ -324,7 +324,7 @@ def show
     )
 
     if params[:key] == 'artist'
-      @project_image = CDO.studio_url "/v3/files/#{@view_options['channel']}/_share_image.png", 'https:'
+      @project_image = CDO.studio_url "/v3/files/#{@view_options['channel']}/.metadata/thumbnail.png", 'https:'
```

Comment:
can we combine this with line 331?

Reply:
Yes, but for two reasons, I'd prefer to keep this separate
1. This is a workaround until we can get the thumbnail+border working
2. I'm trying to keep this very scoped and targeted